### PR TITLE
Deduplication of launch scripts and support for re-defining paths

### PR DESCRIPTION
**Pull request by vstax**
Implementation of https://github.com/leo-project/leofs/issues/638, except for package changes and actual switch to non-privileged user.

### CODE REVIEW

File: rel/common/launch.sh [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+
+# Make sure log directory exists
+mkdir -p $RUNNER_LOG_DIR
+# Identify the script name
+SCRIPT=`basename $0`
+
+help () {
+    echo "Usage: $SCRIPT [-type leo_manager|leo_gateway|leo_storage] {start|stop|ping|remote_console}"
+    echo "Script type is picked from its name, but can be overriden with -type option"
+    echo "leo_manager additionally supports commands {dump-mnesia-data|load-mnesia-data|clear-history-data}"
```
Comment (mocchira) [inline rel/common/launch.sh]: It's not your fault though please remove clear-history-data as it got obsoleted in the past.

File: rel/common/launch.sh [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+
+# Setup command to control the node
+NODETOOL="$ERTS_PATH/escript $ERTS_PATH/nodetool $NAME_ARG $COOKIE_ARG"
+NODETOOL_LITE="$ERTS_PATH/escript $ERTS_PATH/nodetool"
+
+# Setup remote shell command to control node
+REMSH="$ERTS_PATH/erl $REMSH_NAME_ARG $REMSH_REMSH_ARG $COOKIE_ARG"
+
+case "$1" in
+    dump-mnesia-data|load-mnesia-data|clear-history-data)
```
Comment (mocchira) [inline rel/common/launch.sh]: Remove clear-history-data

File: rel/common/launch.sh [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+        if [ "$NODE_TYPE" != leo_manager ]; then
+            echo "Only leo_manager supports {dump-mnesia-data|load-mnesia-data|clear-history-data} commands"
```
Comment (mocchira) [inline rel/common/launch.sh]: Remove clear-history-data

File: rel/common/launch.sh [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+        ;;
+    clear-history-data)
+        ## Clear records of history-table
+        $NODETOOL rpc mnesia clear-history-data []
+
+        ES=$?
+        if [ "$ES" -ne 0 ]; then
+            exit $ES
+        fi
+        ;;
```
Comment (mocchira) [inline rel/common/launch.sh]: Remove the whole block of clear-history-data

**Comment by vstax** [comment]
@mocchira Here is fixed version

**Pull request merged**